releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -5,8 +5,13 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
 ### v5.1.0 - January, 07 2022
-* Added support for sovereign clouds such as Azure Government and Azure China.
-* Updated task input from **App Configuration Name** to **App Configuration Endpoint**.
+* **Breaking change**: 
+With the added support of sovereign clouds such as Azure Government and Azure China, the task inputs were updated.
+  **Before**: 
```

Comment:
An empty line is needed before this or it will be merged in the same line with the bullet point.